<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>201</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SpotiQuit 0.1.0</string>
  </property>
  <property name="windowIcon">
   <iconset theme="pda">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(15, 37, 35);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Restart Spotify and resume playback</string>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(225, 225, 225);
font: 12pt &quot;Times New Roman&quot;;
</string>
        </property>
        <property name="text">
         <string>Restart Spotify!</string>
        </property>
        <property name="shortcut">
         <string>Ctrl+R</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="toolTip">
         <string>Current status</string>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(216, 184, 49);
font: 700 11pt &quot;Arial&quot;;
</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Ready&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="toolTip">
       <string>Choose the button that will be used to resume playback.</string>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(216, 184, 49);
font: 700 9pt &quot;Arial&quot;;</string>
      </property>
      <property name="title">
       <string>Play key</string>
      </property>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>161</width>
         <height>50</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="1" column="0">
         <widget class="QRadioButton" name="radioButton">
          <property name="styleSheet">
           <string notr="true">color: rgb(216, 184, 49);
font: 700 9pt &quot;Arial&quot;;</string>
          </property>
          <property name="text">
           <string>Play / Pause Media Key</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QRadioButton" name="radioButton_2">
          <property name="styleSheet">
           <string notr="true">color: rgb(216, 184, 49);
font: 700 9pt &quot;Arial&quot;;</string>
          </property>
          <property name="text">
           <string>Spacebar</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QCheckBox" name="checkBox">
      <property name="toolTip">
       <string>Minimises the Spotify window after resuming playback.</string>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(216, 184, 49);
font: 700 9pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>Minimise</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>pushButton</tabstop>
  <tabstop>radioButton_2</tabstop>
  <tabstop>radioButton</tabstop>
  <tabstop>checkBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
